### ISSUE TITLE
Not working anymore with Pillow==3.4.0

### ISSUE THREAD

**Issue by lzanuz** [mbi/django-simple-captcha#111]
Error:  Size must be a tuple.

views.py in captcha_image

Line 93:         charimage = Image.new('L', getsize(font, ' %s ' % char), '#000000')

**Comment by Real-Gecko** [comment]
Confirming:
```
Internal Server Error: /captcha/image/a995dcbbd074233c3a915fb8901b0fe4785a060b/
Traceback (most recent call last):
  File "~/venv/lib/python3.4/site-packages/django/core/handlers/exception.py", line 39, in inner
    response = get_response(request)
  File "~/venv/lib/python3.4/site-packages/django/core/handlers/base.py", line 249, in _legacy_get_response
    response = self._get_response(request)
  File "~/venv/lib/python3.4/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "~/venv/lib/python3.4/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "~/venv/lib/python3.4/site-packages/captcha/views.py", line 93, in captcha_image
    charimage = Image.new('L', getsize(font, ' %s ' % char), '#000000')
  File "~/venv/lib/python3.4/site-packages/PIL/Image.py", line 2021, in new
    _check_size(size)
  File "~/venv/lib/python3.4/site-packages/PIL/Image.py", line 1997, in _check_size
    raise ValueError("Size must be a tuple")
```

**Comment by mbi** [comment]
Thanks for the heads-up, this should be fixed in 0.5.3 which I just pushed to pypi.
